lib/src/message.dart

```diff
@@ -134,17 +143,48 @@ abstract class Message {
 ///
```

Comment:
maybe mention in the doc comment (1 line above) that this also adds info about the content-length?